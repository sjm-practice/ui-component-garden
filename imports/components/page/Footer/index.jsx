import React from "react";
import BaseFooter from "./styles";

const Footer = () => (
  <BaseFooter>
    <img src="images/carrot_icon.png" alt="Componay Logo" />
    <span>Copyright 2018 Eat. Drink. Healthy.</span>
    <span>All rights reserved.</span>
  </BaseFooter>
);

export default Footer;
