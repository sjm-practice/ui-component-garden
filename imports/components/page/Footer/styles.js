import styled from "styled-components";

/* Footer Styling */
const BaseFooter = styled.footer`
  background-color: #f28020;
  color: #fff;
  font-size: 0.8em;
  padding: 1.25em 0;
  display: flex;
  flex-wrap: wrap;
  align-items: center;

  img {
    margin-left: 10%;
    margin-right: 3px;
  }

  span {
    margin: 4px;
  }

  @media all and (max-width: 599px) {
    & {
      flex-direction: column;
    }

    img {
      margin: 0 0 4px 0;
    }
  }
`;

export default BaseFooter;
