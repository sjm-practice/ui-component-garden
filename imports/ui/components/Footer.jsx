import React from "react";

const Footer = () => (
  <footer className="footer-container">
    <img src="images/carrot_icon.png" alt="Componay Logo" />
    <span>Copyright 2018 Eat. Drink. Healthy.</span>
    <span>All rights reserved.</span>
  </footer>
);

export default Footer;
