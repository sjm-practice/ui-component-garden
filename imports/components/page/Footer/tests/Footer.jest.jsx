/* eslint-env jest */
/* eslint-disable func-names, prefer-arrow-callback, no-unused-expressions */
/* eslint-disable import/no-extraneous-dependencies */
import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";
import Footer from "..";

describe("<Footer />", () => {
  it("should match render snapshot", () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
