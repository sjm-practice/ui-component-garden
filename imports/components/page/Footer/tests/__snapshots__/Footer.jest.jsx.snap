// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should match render snapshot 1`] = `
<footer
  className="footer-container"
>
  <img
    alt="Componay Logo"
    src="images/carrot_icon.png"
  />
  <span>
    Copyright 2018 Eat. Drink. Healthy.
  </span>
  <span>
    All rights reserved.
  </span>
</footer>
`;
